cmd/balance-worker/main.go

```diff
@@ -211,14 +210,42 @@ func main() {
 	}
 
 	// Dependencies: postgresql
-	pgClients, err := initPGClients(conf.Postgres)
+
+	// Initialize Postgres driver
+	var postgresDriver *pgdriver.Driver
+	postgresDriver, err = pgdriver.NewPostgresDriver(conf.Postgres.URL,
+		pgdriver.WithContext(ctx),
+		pgdriver.WithTracerProvider(otelTracerProvider),
+		pgdriver.WithMeterProvider(otelMeterProvider),
+	)
 	if err != nil {
-		logger.Error("failed to initialize postgres clients", "error", err)
+		logger.Error("failed to initialize postgres driver", "error", err)
 		os.Exit(1)
 	}
-	defer pgClients.driver.Close()
 
-	logger.Info("Postgres clients initialized")
+	defer func() {
+		if err = postgresDriver.Close(); err != nil {
+			logger.Error("failed to close postgres driver", "error", err)
+		}
+	}()
+
+	// Initialize Ent driver
+	entPostgresDriver := entdriver.NewEntPostgresDriver(postgresDriver.DB())
+	defer func() {
+		if err = entPostgresDriver.Close(); err != nil {
+			logger.Error("failed to close ent driver", "error", err)
+		}
+	}()
+
+	entPostgresClient := entPostgresDriver.Client()
```

Comment:
Isn't this just an ent client at this point? I mean, it's not postgres specific, is it?

Reply:
I do not think it makes any difference, but will rename it.